src/helpers/regex.helper.ts

```diff
@@ -1,6 +1,15 @@
 const socialNetworksUrlRegex = new RegExp(
   '^(https?://)?(www.)?(facebook.com|fb.me|twitter.com|vxtwitter.com|instagram.com|linkedin.com|youtube.com|youtu.be|pinterest.com|snapchat.com|tiktok.com)/[a-zA-Z0-9.-/?=&#_]+$',
 );
+const punctuationRegex = new RegExp(/[.,!?]/g);
+const emojiRegex = new RegExp(/(\p{Extended_Pictographic}|\p{Emoji_Component})/gu);
+const quoiDetectorRegex = new RegExp(/\b\s*[q][u][o][i]\s*$/i);
```

Comment:
While `punctuationRegex` and `emojiRegex` can make sense as global helpers, I don't think there's a lot of use-cases for other modules to detect sentences that ends with `quoi`.

Can you move this inside your module helper please ?